.github/ISSUE_TEMPLATE/30-impl-proposal.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Implementation proposal
+about: Request or propose an implementation that solves one or more use cases.
```

Comment:
"Implementation proposal" seems like an odd name given that we usually use "Implementation" to mean "something that implements this spec" whereas it sounds like this template is actually for suggestions to add or change something in the spec?

Maybe "Specification change proposal" would be clearer.

Reply:
done